lib/src/features/bmi/presentation/pages/BMICalculatorScreen.dart

```diff
@@ -36,9 +36,12 @@ class BMICalculatorScreen extends HookConsumerWidget {
     final heightUnits =
         useState<HeightUnitsPickerOptions>(HeightUnitsPickerOptions.In);
     final weight = useState<int>(65);
-    final age = useState<int>(27);
     final isForSelf = useState(true);
     final userAsync = ref.watch(userProvider);
+    var age = userAsync.whenData((user) {
```

Comment:
Yes though when I have used it it doesn't work as expected. Using the getAge method from the AuthRepository will require me to resolve a Future variable so I had tried using the useEffect hook however it was not updating the UI when the widget is first initialised forcing the user to have to click the BMI for myself option for the correct age to be rendered yet the default value is already the user's current age set using the getAge method. I believe my current implementation still works unless you have an idea of another work around of using the getAge method. 

Also I thought it would be easier to just use the userProvider since it was already instatiated for this scope and it was the same that we were using for the gender section

Reply:
It should be applicable when calculating for oneself otherwise it should be even be 0.